web/gui-v2/src/components/DetailViewIntro.jsx

```diff
@@ -37,26 +44,34 @@ const styles = {
       padding: 0.5rem;
     }
   `,
+  buttonWrapper: css`
+    display: flex;
+    margin-top: 1rem !important;
+
+    button {
+      margin-left: auto;
+    }
+  `,
 };
 
 const DetailViewIntro = ({
   companyId,
   data,
 }) => {
+  const [dialogOpen, setDialogOpen] = useState(false);
 
-  const metadataColumns = [
-    { display_name: "", key: "title" },
-    { display_name: "", key: "data", align: "right" },
+  const metadata = [
+    { title: "Country", value: data.country },
+    { title: "Sector", value: "--TODO--" },
+    { title: "Website", value: data.website ? <ExternalLink href={data.website}>{data.website}</ExternalLink> : undefined },
```

Comment:
I looked at the definition of TwoColumnTable and don't see that the undefined case is explicitly handled (may have missed it). So I'm assuming it will show up blank if no website - can we maybe do something like "None found" instead?

Reply:
I've added a "None found" note in this case